app/views/devise/passwords/edit.html.slim

```diff
@@ -14,6 +14,9 @@
         = f.input :reset_password_token, as: :hidden
         = f.full_error :reset_password_token
         = f.input :password, label: 'New password', required: true, autofocus: true
+        = render 'devise/shared/password_strength'
         = f.input :password_confirmation, label: 'Confirm your new password', required: true
         = f.button :submit, 'Change my password'
       = render 'devise/shared/links'
+
+== javascript_include_tag 'misc/pw-strength'
```

Comment:
totally agree about the pipeline, we def want to limit the requests, and everything in the application.js manifest gets bundled into 1 file (which includes the app/utils.js; it just appears as its own file when working in dev). however, re: zxcvbn, i think we want to separate this into its own file because of its size (this is what https://github.com/dropbox/zxcvbn recommends too)